google/cloud/storage/internal/list_objects_request_test.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/list_objects_request.h"
+#include <gmock/gmock.h>
+
+namespace gcs = google::cloud::storage;
+using namespace google::cloud::storage::internal;
```

Comment:
Rather than using the entire namespace, can we only have `using` statements for elements from this namespace that we need?

Reply:
N/A now that we included the test in the right namespace.